alembic/versions/0003_response_event.py

```
@@ -47,9 +47,11 @@
 """
 
 _TRIGGER_SQL = """
+# #43：FOR EACH ROW——分区表上语句级触发器只覆盖父表语句级操作，行级路径
```

Ruff (0.16.1)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)